### ISSUE TITLE
TypeError: 'list' object cannot be interpreted as an integer

### ISSUE THREAD

**Issue by schreon** [ElementAI/picard#75]
I pulled the newest version of picard and now I'm getting the following error whenever I send a request to the REST api:

```
03/14/2022 08:32:03 - ERROR - uvicorn.error -   Exception in ASGI application
Traceback (most recent call last):
  File "/opt/conda/lib/python3.7/site-packages/uvicorn/protocols/http/h11_impl.py", line 366, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/opt/conda/lib/python3.7/site-packages/uvicorn/middleware/proxy_headers.py", line 75, in __call__
    return await self.app(scope, receive, send)
  File "/opt/conda/lib/python3.7/site-packages/fastapi/applications.py", line 259, in __call__
    await super().__call__(scope, receive, send)
  File "/opt/conda/lib/python3.7/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/opt/conda/lib/python3.7/site-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc
  File "/opt/conda/lib/python3.7/site-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/opt/conda/lib/python3.7/site-packages/starlette/exceptions.py", line 82, in __call__
    raise exc
  File "/opt/conda/lib/python3.7/site-packages/starlette/exceptions.py", line 71, in __call__
    await self.app(scope, receive, sender)
  File "/opt/conda/lib/python3.7/site-packages/fastapi/middleware/asyncexitstack.py", line 21, in __call__
    raise e
  File "/opt/conda/lib/python3.7/site-packages/fastapi/middleware/asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
  File "/opt/conda/lib/python3.7/site-packages/starlette/routing.py", line 656, in __call__
    await route.handle(scope, receive, send)
  File "/opt/conda/lib/python3.7/site-packages/starlette/routing.py", line 259, in handle
    await self.app(scope, receive, send)
  File "/opt/conda/lib/python3.7/site-packages/starlette/routing.py", line 61, in app
    response = await func(request)
  File "/opt/conda/lib/python3.7/site-packages/fastapi/routing.py", line 228, in app
    dependant=dependant, values=values, is_coroutine=is_coroutine
  File "/opt/conda/lib/python3.7/site-packages/fastapi/routing.py", line 162, in run_endpoint_function
    return await run_in_threadpool(dependant.call, **values)
  File "/opt/conda/lib/python3.7/site-packages/starlette/concurrency.py", line 39, in run_in_threadpool
    return await anyio.to_thread.run_sync(func, *args)
  File "/opt/conda/lib/python3.7/site-packages/anyio/to_thread.py", line 29, in run_sync
    limiter=limiter)
  File "/opt/conda/lib/python3.7/site-packages/anyio/_backends/_asyncio.py", line 818, in run_sync_in_worker_thread
    return await future
  File "/opt/conda/lib/python3.7/site-packages/anyio/_backends/_asyncio.py", line 754, in run
    result = context.run(func, *args)
  File "seq2seq/serve_seq2seq.py", line 126, in ask
    outputs = pipe(inputs=Text2SQLInput(utterance=question, db_id=db_id))
  File "/app/seq2seq/utils/pipeline.py", line 76, in __call__
    result = super().__call__(inputs, **kwargs)
  File "/opt/conda/lib/python3.7/site-packages/transformers/pipelines/text2text_generation.py", line 138, in __call__
    result = super().__call__(*args, **kwargs)
  File "/opt/conda/lib/python3.7/site-packages/transformers/pipelines/base.py", line 1027, in __call__
    return self.run_single(inputs, preprocess_params, forward_params, postprocess_params)
  File "/opt/conda/lib/python3.7/site-packages/transformers/pipelines/base.py", line 1035, in run_single
    outputs = self.postprocess(model_outputs, **postprocess_params)
  File "/app/seq2seq/utils/pipeline.py", line 138, in postprocess
    clean_up_tokenization_spaces=clean_up_tokenization_spaces,
  File "/opt/conda/lib/python3.7/site-packages/transformers/tokenization_utils_base.py", line 3316, in decode
    **kwargs,
  File "/opt/conda/lib/python3.7/site-packages/transformers/tokenization_utils_fast.py", line 548, in _decode
    text = self._tokenizer.decode(token_ids, skip_special_tokens=skip_special_tokens)
TypeError: 'list' object cannot be interpreted as an integer

```

Is this a mistake on my end or is there a general problem with the newer images?

**Comment by tscholak** [comment]
Hi @schreon,
Thanks for reporting, let me have a look!

**Comment by tscholak** [comment]
Upstream changed, https://github.com/huggingface/transformers/commit/06107541d3df39e3d9685e64c3b942d9aed06d75, therefore the exception.

**Comment by udaramrdc** [comment]
Hi @tscholak,
I am also facing the same issue, 
Could you please tell me, how can i fix this issue?
Thanks in advance

**Comment by tscholak** [comment]
Hi @udaramrdc, please make sure you're using the latest Docker image(s) that contain the fixes.

**Comment by DreamerDeo** [comment]
Hi @schreon and @tscholak , I met the same problem here. In the latest docker `tscholak/text-to-sql-eval:cache`, the `transformer` version is 4.17.0. I solve this by downgrade it to `4.11.3`.

**Comment by tscholak** [comment]
Hi @DreamerDeo,
It's more likely that your local images are outdated, because I fixed this issue with 4.17.0 a month ago.

**Comment by DreamerDeo** [comment]
@tscholak Sorry, you are right. I have updated my images and it works now. :)
